lua/neo-tree/utils.lua

```diff
@@ -349,13 +351,17 @@ M.open_file = function(state, path, open_cmd)
         -- Neo-tree must be the only window, restore it's status as a sidebar
         local winid = vim.api.nvim_get_current_win()
         local width = M.get_value(state, "window.width", 40)
-        vim.cmd("vsplit " .. escaped_path)
+        result, err = pcall(vim.cmd, "vsplit " .. escaped_path)
         vim.api.nvim_win_set_width(winid, width)
       else
-        vim.cmd(open_cmd .. " " .. escaped_path)
+        result, err = pcall(vim.cmd, open_cmd .. " " .. escaped_path)
       end
     end
-    events.fire_event(events.FILE_OPENED, path)
+    if (result or err == "Vim(edit):E325: ATTENTION") then
+      events.fire_event(events.FILE_OPENED, path)
+    else
+      print(err)
```

Comment:
There is an internal logging mechanism that should be used here, something like:
```suggestion
      log.error("Error opening file:", err)
```